src/ray/raylet/object_manager_integration_test.cc

```diff
@@ -74,8 +74,8 @@ class TestObjectManagerBase : public ::testing::Test {
         GetNodeManagerConfig("raylet_2", store_sock_2), om_config_2, gcs_client_2));
 
     // connect to stores.
-    ARROW_CHECK_OK(client1.Connect(store_sock_1, ""));
-    ARROW_CHECK_OK(client2.Connect(store_sock_2, ""));
+    ARROW_CHECK_OK(client1.Connect(store_sock_1));
+    ARROW_CHECK_OK(client2.Connect(store_sock_2));
```

Comment:
In this file, it mixes with `RAY_CHECK` and `ARROW_CHECK_OK`. Can we change it to `RAY_CHECK_OK` to unify them? There are some other test files that have the same problem.

Reply:
The call to `client.Connect` returns an `arrow::Status`, which is why we call `ARROW_CHECK_OK` (since the plasma client is part of Arrow). We use `RAY_CHECK_OK` when we have a `ray::Status` object.